src/App.js

```diff
@@ -6,6 +6,18 @@ import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
+import { PostList } from './components/PostList';
+
+const postsWithComments = posts.map(item => ({
+  ...item,
+  comment: comments.filter(comment => comment.postId === item.id),
+}));
```

Comment:
this definitely should work 

```suggestion
const preparedPosts = posts.map(post => ({
  ...item,
  comment: comments.filter(comment => comment.postId === post.id),
  user: users.find(user => user.id === post.userId),
}));
```

Reply:
Oh, its grate idea!